lib/jsxgettext.js

```diff
@@ -84,12 +84,28 @@ function getTranslatable(node, options) {
   if (options.keyword.indexOf(funcName) === -1)
     return false;
 
+  // Domain is not used during extraction, so ignore it
+  if(funcName.substr(0, 1) === "d"){
+    funcName = funcName.substr(1, funcName.length);
+    node.arguments.splice(0, 1);
+    arg = node.arguments[0];
+  }
+
+  // Always add the funcName as last element in return array
+  // If the gettext function's name starts with "np" (i.e. npgettext or np_) and its 3 arguments are strings, we regard it as context  
```

Comment:
Is there documentation around this for existing gettext toolchain? If yes, it would be great to reference that here. If not, I'd like to hear more about your reasoning :)